zebrad/tests/common/config.rs

```diff
@@ -80,3 +84,15 @@ pub fn testdir() -> Result<TempDir> {
         .tempdir()
         .map_err(Into::into)
 }
+
+/// Get stored config path
+pub fn stored_config_path() -> PathBuf {
+    Path::new(env!("CARGO_BIN_EXE_zebrad"))
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap()
+        .join("zebrad/tests/common/config.toml")
```

Comment:
This can be fragile, not to mention ugly. Open for a better way to do it.

Reply:
Here's one way to avoid using the file path:
1. Use include_str!() to turn it into a string variable: https://doc.rust-lang.org/std/macro.include_str.html
2. Parse the string as a config, if parsing fails, fail the test
3. Return the config from this function
4. Launch zebrad with the config using zebrad.with_config()

You could skip parsing the config, but then you'd have to set up the temp path and process arguments yourself.

Reply:
(This is pretty much what `cargo insta` does with its snapshots, or it handles the different paths internally.)

Reply:
Hard to follow what to do and to me, unnecessary. I did some changes to this function but feel free to  change it to what you want.